src/util/BrowserFeatures.js

```diff
@@ -55,7 +55,7 @@ define(function () {
   };
 
   fn.isIE = function () {
-    return /msie/i.test(navigator.userAgent);
+    return /msie/i.test(navigator.userAgent) || /trident/i.test(navigator.userAgent);
```

Comment:
@haishengwu-okta done.
@lboyette-okta There are a couple of ways to do this, but checking for the useragent string seems to be the most commonly used, so I went with it.